coxeter/shapes/polyhedron.py

```diff
@@ -356,9 +356,36 @@ def vertices(self):
 
     @property
     def faces(self):
-        """list(:class:`numpy.ndarray`): Get the polyhedron's faces."""
+        """list(:class:`numpy.ndarray`): Get the polyhedron's faces.
+
+        Results returned as vertex index lists.
+        """
         return self._faces
 
+    @property
+    def edges(self):
+        """list(:class:`numpy.ndarray`): Get the polyhedron's edges.
+
+        Results returned as vertex index pairs.
+        """
+        edges = []
+        for face in self.faces:
+            [
+                edges.append((i, j))
+                for i, j in zip(np.append(face, face[0]), np.append(face[1:], face[0]))
+                if (j, i) not in edges
+            ]
+        return edges
+
+    @property
+    def get_edge_vectors(self):
```

Comment:
We don't name properties with `get_*` prefixes, only functions. Properties should "feel like" accessing an attribute of an object.
```suggestion
    def edge_vectors(self):
```